apps/meteor/app/tokenpass/client/hooks/useTokenPassAuth.tsx

```diff
@@ -23,11 +22,14 @@ const config: OauthConfig = {
 
 const Tokenpass = new CustomOAuth('tokenpass', config);
 
-Meteor.startup(() => {
-	Tracker.autorun(() => {
-		if (settings.get('API_Tokenpass_URL')) {
-			config.serverURL = settings.get('API_Tokenpass_URL');
-			Tokenpass.configure(config);
+export const useTokenPassAuth = () => {
+	const setting = useSetting('API_Tokenpass_URL');
```

Comment:
Could you specify the type here and avoid the cast on L32?
```suggestion
	const setting = useSetting<string>('API_Tokenpass_URL');
```